src/index.js

```diff
@@ -0,0 +1,16 @@
+import './style.css';
+import { addTodo, deleteAll, renderTodos } from './functions.js';
+
+document.querySelector('.text-input').addEventListener('keypress', (e) => {
```

Comment:
Good work so far, Kindly use a meaningful word instead of single letter for function argument.